### ISSUE TITLE
Calendar: Today/Selected date is not read by screen reader

### ISSUE THREAD

**Issue by CharlesgdHill** [OfficeDev/office-ui-fabric-react#8305]
"Today" and selected date isn't read by screen reader in calendar upon navigation to calendar:
Today is selected and highlighted on the calendar, but not read out by screen reader when calendar is tabbed to, resulting in lost information for users using screen reader.

Please provide a reproduction of the bug in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/6.155.0/#/examples/calendar

Repro Steps: Tab to a calendar with screen reader enabled.

#### Actual behavior:
 Screen reader reads "table has 7 rows, 7 columns (etc)"
#### Expected behavior:
 Screen reader should read out the currently selected date, and the highlighted "today" date.

Requested priority: Normal

**Comment by KevinTCoughlin** [comment]
@lorejoh12 does this a11y problem surface in both calendar controls? What would you advise we do to address the above issue?

**Comment by KevinTCoughlin** [comment]
I spoke with @lorejoh12 today who recommended we put Calendar related issues under the date-time project so that it receives proper attention by the feature crew.

**Comment by lorejoh12** [comment]
I think we can fix this by adding the information for "today date Nov 11 2019, selected date Nov 11 2019" directly to the aria-label at the root, so it only gets read the first time.

**Comment by lorejoh12** [comment]
For some reason the PR merging didn't auto close the issues... closing, this should be fixed now.
